src/web/routes/figma/figma-router.ts

```diff
@@ -21,8 +21,8 @@ import { requestSchemaValidationMiddleware } from '../../middleware';
 import { figmaWebhookAuthMiddleware } from '../../middleware/figma/figma-webhook-auth-middleware';
 
 const AUTH_RESOURCE_BASE_PATH = '/static/auth-result';
-export const SUCCESS_PAGE_URL = `${AUTH_RESOURCE_BASE_PATH}?success=true`;
-export const FAILURE_PAGE_URL = `${AUTH_RESOURCE_BASE_PATH}?success=false`;
+export const SUCCESS_PAGE_URL = `${AUTH_RESOURCE_BASE_PATH}/authSuccess?success=true`;
```

Comment:
**nitpick:** Please, remove `success` query parameter as it is not needed anymore.

Also, please, consider:

1. Removing the `AUTH_RESOURCE_BASE_PATH` variable above.
2. Renaming `authSuccess` to `auth-success` or just `success` to match the current naming convention for static resources. The same is applicable for `authFailure`.

Long story shory, here is what we could get with all these suggestions applied:

```ts
export const SUCCESS_PAGE_URL = `/static/auth-result/success`;
export const FAILURE_PAGE_URL = `/static/auth-result/failure`;
```